smb_path_tracker/launch/pure_pursuit_tracker.launch

```diff
@@ -0,0 +1,15 @@
+<launch>
+
+	<arg name="odom_topic" default="/tracking_camera/odom/sample"/>
+	<arg name="cmd_vel_topic" default="/control/joy_teleop/cmd_vel"/>
```

Comment:
we have a muxer that "fuses" different topics with twist commands depending on their priority. 
So this tracker should publish to its own topic (e.g. `/control/pure_pursuit/cmd_vel`) and the [muxer adjusted accordingly](https://github.com/ETHZ-RobotX/smb_common/blob/devel/smb_control/config/twist_mux.yaml).

Reply:
Thanks for the suggestion. I have now changed it